ipol_demo/modules/blobs/blobs.py

```diff
@@ -159,12 +159,10 @@ def get_db_connection():
     try:
         connection = lite.connect(settings.database_file)
         yield connection
-    except IPOLBlobsDataBaseError:
-        log.exception("DB error could not get a connection")
-        print("DB error could not get a connection")
-    except Error as e:
-        log.exception(f"Error connectin to database: {e}")
-        print(f"Error connecting to database: {e}")
+    except IPOLBlobsDataBaseError as ex:
+        log.exception(f"DB error could not get a connection: {ex}")
```

Comment:
Exceptions (with stacktrace) are already logged properly when using `Logger.exception`, so no need to capture `ex` here.

```python
import logging
logger = logging.getLogger()
try:
    raise ValueError("something")
except Exception:
    logger.exception("test")
```

Reply:
Do you mean all of the log.exception in the entire file or just for IPOLBlobsDataBaseError?

Reply:
All of them, this is just an example.